x-pack/plugins/monitoring/public/components/elasticsearch/shard_activity/parse_props.js

```diff
@@ -33,6 +33,8 @@ export const parseProps = (props) => {
     primary: isPrimary,
     start_time_in_millis: startTimeInMillis,
     total_time_in_millis: totalTimeInMillis,
+    start_time: { ms: mbStartTimeInMillis },
```

Comment:
I'll add in the guards where they are needed but I feel that it's out of scope to make the API hide these details simply because we don't do this consistently and understanding where we use this data is a bit larger of a task.